service_mesh/v2x/ossm-security.adoc

```diff
@@ -36,4 +36,22 @@ include::modules/ossm-cert-manage-add-cert-key.adoc[leveloffset=+1]
 
 include::modules/ossm-cert-manage-verify-cert.adoc[leveloffset=+1]
 
-include::modules/ossm-cert-cleanup.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/ossm-cert-cleanup.adoc[leveloffset=+1]
+
+include::modules/ossm-cert-manager-integration-istio.adoc[leveloffset=+1]
+
+include::modules/ossm-cert-manager-install.adoc[leveloffset=+2]
+
+include::modules/ossm-cert-manager-verify.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_cert-manager-operator-red-hat-openshift"]
+== Additional resources 
+
+For information on how to install the cert-manager Operator for Red Hat OpenShift, see:
```

Comment:
```suggestion
For information about how to install the cert-manager Operator for {product-title}, see:
```
- use "information about' rather than "information on"; "on" can mean different things.
- use an attribute for the OpenShift product name